reference/configuration/framework.rst

```diff
@@ -3082,6 +3082,81 @@ name
 
 Name of the lock you want to create.
 
+semaphore
+~~~~~~~~~
+
+.. versionadded:: 6.1
+
+    The ``semaphore`` option was introduced in Symfony 6.1.
+
+**type**: ``string`` | ``array``
+
+The default semaphore adapter. Store's DSN are also allowed.
+
+.. _reference-lock-enabled:
+
+enabled
+.......
+
+**type**: ``boolean`` **default**: ``true``
+
+Whether to enable the support for semaphore or not. This setting is
+automatically set to ``true`` when one of the child settings is configured.
+
+.. _reference-lock-resources:
+
+resources
+.........
+
+**type**: ``array``
+
+A list of semaphore stores to be created by the framework extension.
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # config/packages/semaphore.yaml
+        framework:
+            semaphore: '%env(SEMAPHORE_DSN)%'
```

Comment:
```suggestion
            semaphore:
                resources:
                    default: '%env(SEMAPHORE_DSN)%'
```

Reply:
Both are correct, but the first one (`semaphore: '%env(SEMAPHORE_DSN)%'`) is easiest to read IMHO and consistent with `lock` documentation